app/controllers/workflows_controller.rb

```diff
@@ -1,12 +1,16 @@
 class WorkflowsController < ApplicationController
+  responders :flash
+
   def index
-    @workflows = policy_scope(Workflow).all
+    @workflows = policy_scope(Workflow).all.sort_by(&:id)
+    @workflow = Workflow.new
     respond_with @workflows
   end
 
   def show
     authorize workflow
-    respond_with workflow
+    heartbeat
```

Comment:
If extracted, this could become `@heartbeat = Heartbeat.new(workflow)` which also gets rid of calling this method just for it's side effect of assigning an instance variable.